src/Mappers/InventoryMapper.php

```diff
@@ -1,24 +1,31 @@
 <?php
+
 /**
- * @copyright 2019 Wayfair LLC - All rights reserved
+ * @copyright 2020 Wayfair LLC - All rights reserved
  */
 
 namespace Wayfair\Mappers;
 
 use Plenty\Modules\Item\VariationStock\Contracts\VariationStockRepositoryContract;
+use Wayfair\Core\Contracts\LoggerContract;
 use Wayfair\Core\Dto\Inventory\RequestDTO;
 use Wayfair\Core\Helpers\AbstractConfigHelper;
+use Wayfair\Helpers\TranslationHelper;
 use Wayfair\Repositories\KeyValueRepository;
 use Wayfair\Repositories\WarehouseSupplierRepository;
 
-class InventoryMapper {
+class InventoryMapper
+{
+  const LOG_KEY_STOCK_MISSING_WAREHOUSE = 'stockMissingWarehouse';
+  const LOG_KEY_NO_SUPPLIER_ID_ASSIGNED_TO_WAREHOUSE = 'noSupplierIDForWarehouse';
 
   /**
    * @param $mainVariationId
    *
    * @return mixed
    */
-  public function getAvailableDate($mainVariationId) {
+  public function getAvailableDate($mainVariationId)
```

Comment:
Might be good to add a todo or a ticket for refining/verifying this logic

Reply:
yeah a ticket is a good idea here.